packages/channel-provider/src/logger.ts

```diff
@@ -36,12 +37,21 @@ const browser: any = IS_BROWSER_CONTEXT
   : undefined;
 
 const prettyPrint = LOG_TO_CONSOLE ? {translateTime: true} : false;
-// When logging, we default to 'info', as most logs happen at this level.
-// Some very large classes are serialized at the 'trace' level
-// We probably don't want these logged to the console, but strictly enabling this
-// in the browser might sometimes be helpful
-// eslint-disable-next-line no-undef
-export const LOG_LEVEL = LOG_TO_FILE || LOG_TO_CONSOLE ? process.env.LOG_LEVEL || 'info' : 'silent';
 
-const opts = {name, prettyPrint, browser, level: LOG_LEVEL};
+const ADD_LOGS = LOG_TO_FILE || LOG_TO_CONSOLE;
+const LOG_LEVEL = ADD_LOGS ? (process.env.LOG_LEVEL ? process.env.LOG_LEVEL : 'info') : 'silent';
+const level = window.localStorage.LOG_LEVEL ?? LOG_LEVEL;
```

Comment:
Is local storage used so that the log level persists across page refreshes?

Reply:
Actually, I think it's fine as is.